### PR TITLE
Auth0 spike

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.

AB#User Story 641: SPIKE: Auth0

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx [hunk 1]
```diff
@@ -44,17 +50,21 @@ class _App extends App<AppWithStore> {
...
         const {Component, pageProps, store} = this.props
+        const props = {
+            ...pageProps,
+            user: this.state,
+        }
         return (
             <>
                 <Head>
-                    <title>Amido stacks</title>
+                    <title>Amido stacks SSR</title>
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx:60]: Can we change this to `Amido Stacks SSR` ? Sorry! Captilisation.

File: packages/template-cli/templates/src/ssr/README.md [hunk 2]
```diff
@@ -1,286 +0,0 @@
-# Web Application
```
Comment (estherthetesteramido) [inline packages/template-cli/templates/src/ssr/README.md:1]: should this be deleted?

File: packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js [hunk 3]
```diff
@@ -7,4 +7,10 @@ module.exports = {
     LOG_LEVEL: process.env.LOG_LEVEL,
     POD_NAME: process.env.POD_NAME,
     APPINSIGHTS_KEY: process.env.APPINSIGHTS_INSTRUMENTATIONKEY,
+
+    AUTH0_BASE_URL: `${process.env.APP_BASE_URL}:${process.env.PORT}`,
+    AUTH0_CLIENT_ID: process.env.AUTH0_CLIENT_ID,
+    AUTH0_DOMAIN: process.env.AUTH0_DOMAIN,
+    AUTH0_CLIENT_SECRET: process.env.AUTH0_CLIENT_SECRET,
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js:14]: Should the client secret be public? Or should it be part of the server runtime?

Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js:14]: Yes. Good shout. I have seperated them out now.

File: packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx (hunk 1 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx]: Done

**Pull request merged**